### DISCUSSION TITLE
Admin's

### DISCUSSION THREAD

**Discussion by Azoahai** [IBAX-io/go-ibax#1972] [Q&A] [answered]
esteemed IBAX board and management. First of all, I wish you good luck for your success and successful project. I wanted to say that this project is much more than what was thought and ideas. At first I had no knowledge and when I am a new crypto user, I participate in the airdrop by watching videos. I would go wherever. I joined your project in a video and I have come so far. Looking back at this time, this is the best project going forward, IBAX. We've been here from the very beginning, we came with the developers and we number about 6500. The reason why I wrote this article was to wish success and IBAX distribution. 6500 is not a big number, ibax token distribution and I would like all of us to increase the numbers, ibax tokens to be given to everyone. I want to be rich in the second time together. I'm going by writing on behalf of 6500 people. 
            
                Respects...
![IMG_20220408_215744](https://user-images.githubusercontent.com/95991723/162505728-c7928cce-0118-444a-aa3b-3d73b711f008.jpg)

**Comment by yolandadadada** [comment, marked as answer]
Thank you very much for your interest Azoahai, may I ask what country your community members are from? Is it Turkey? If it's Turkey we welcome you and your 6500 brothers to join our official community where we will have a community incentive campaign and when the community reaches a certain size you can contact @IBAXKyle if you want to know more details about becoming a mod!

**Reply by Mdnurunnabi1** [reply]
Vere good

**Comment by Azoahai** [comment]
I was going to prepare CV , resume . We are all already members of the iBax ecosystem. I wanted to tell you so that you are happy. If there is a role for me in this community, I will do it with unconditional love, it will be enough to send a message. With love..
![Uploading Screenshot_2022-04-14-21-40-10-80.jpg…]()
